packages/admin/src/GlobalSearch/DefaultGlobalSearchProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Filament\GlobalSearch;
+
+use Filament\Facades\Filament;
+
+class DefaultGlobalSearchProvider implements Contracts\GlobalSearchProvider
+{
+    public function getResults(string $query): ?array
```

Comment:
Small question, why the return type is nullable? It shouldn't be always an array?

Reply:
The original logic could potentially return null.